config/site_settings.yml

```diff
@@ -1137,6 +1137,9 @@ security:
     default: 'bingbot'
     list_type: compact
   slow_down_crawler_rate: 60
+  content_security_policy:
+    default: "script-src 'unsafe-eval' 'nonce-%{nonce}' %{host}/mini-profiler-resources/ %{host}/service-worker.js %{cdn}/javascripts/;"
```

Comment:
We may want to change this (and include other directives like `image-src`) once I've had a chat with the resident CSP expert within Mozilla.

It currently stands like this because:
* `'unsafe-eval'` - needed to execute Ember
* `'nonce-%{nonce}'` - needed to the vast majority of scripts on the page
* `%{host}/mini-profiler-resources/` - needed to execute the mini profiler, it exists outside of Discourse so we can't nonce it
* `%{host}/service-worker.js` - again something we can't nonce
* `%{cdn}/javscripts/` - needed to load some scripts ace.js loads itself, which is behaviour other libraries might copy